netket/operator/_bose_hubbard/numba.py

```diff
@@ -216,7 +132,7 @@ def _flattened_kernel(  # pragma: no cover
             x_prime = np.empty((batch_size * max_conn, n_sites), dtype=x.dtype)
 
         if pad:
-            x_prime[:, :] = 0
```

Comment:
A few lines above here, can you change the `np.empty` to `np.zeros` for the same reason as above?